<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minder3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="appLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="topPanelLayout">
        <item>
         <layout class="QVBoxLayout" name="view2DLayout">
          <item>
           <spacer name="view2DSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>391</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="topPanelSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>391</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="view3DLayout">
          <item>
           <spacer name="view3DSpacer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>361</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rightPanelLayout">
          <item>
           <layout class="QVBoxLayout" name="objectLayout"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="infoTableLayout"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottomPanelLayout">
        <item>
         <layout class="QVBoxLayout" name="imageTableLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bottomRightPanelLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>830</width>
              <height>194</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabsClosable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="welcomeTab">
             <attribute name="title">
              <string>Welcome: Load and Save</string>
             </attribute>
             <layout class="QHBoxLayout" name="welcomeTabLayout"/>
            </widget>
            <widget class="QWidget" name="visualizationTab">
             <attribute name="title">
              <string>Visualization</string>
             </attribute>
             <layout class="QHBoxLayout" name="visualizationTabLayout"/>
            </widget>
            <widget class="QWidget" name="newTaskTab">
             <attribute name="title">
              <string>New Task</string>
             </attribute>
             <layout class="QHBoxLayout" name="newTaskTabLayout"/>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="statusText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="statusProgressBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>12</height>
               </size>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusViewLogButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string>View Log</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="loadImageMenuItem"/>
    <addaction name="loadSceneMenuItem"/>
    <addaction name="separator"/>
    <addaction name="saveImageMenuItem"/>
    <addaction name="saveOverlayMenuItem"/>
    <addaction name="saveSceneMenuItem"/>
    <addaction name="saveVTKModelsMenuItem"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <action name="loadImageMenuItem">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="saveImageMenuItem">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="saveOverlayMenuItem">
   <property name="text">
    <string>Save Overlay</string>
   </property>
  </action>
  <action name="saveVTKModelsMenuItem">
   <property name="text">
    <string>Save VTK Models</string>
   </property>
  </action>
  <action name="loadSceneMenuItem">
   <property name="text">
    <string>Load Spatial Objects</string>
   </property>
  </action>
  <action name="saveSceneMenuItem">
   <property name="text">
    <string>Save Spatial Objects</string>
   </property>
  </action>
  <action name="savePreProcessedOverlayMenuItem">
   <property name="text">
    <string>Save Pre-Processed Overlay</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="minder3DResources.qrc"/>
 </resources>
 <connections/>
</ui>
